### PR TITLE
More pagination fixes

### PR DESCRIPTION
**Pull request by jerpr**
* Address ExtLoader issues with strict mode on FireFox
* Fix issue when no jQuery Tools $scroller is created

Everything should actually be good this time.

### CODE REVIEW

File: addon-WhoReplied.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<addon addon_id="WhoReplied" title="Who Replied" version_string="1.5.1" version_id="1050100" url="https://xenforo.com/community/resources/who-replied.1537/" install_callback_class="" install_callback_method="" uninstall_callback_class="" uninstall_callback_method="">
+<addon addon_id="WhoReplied" title="Who Replied" version_string="1.5.2" version_id="1050200" url="https://xenforo.com/community/resources/who-replied.1537/" install_callback_class="" install_callback_method="" uninstall_callback_class="" uninstall_callback_method="">
```
Comment (Xon) [inline addon-WhoReplied.xml]: Recent versions of xenforo bump the javascript script-buster string when you install an add-on.

**Comment by Xon** [comment]
Testing this now

**Comment by Xon** [comment]
@Fuhrmann after ~10 hours of testing on https://forums.spacebattles.com this looks fixed now :)

**Comment by Fuhrmann** [comment]
This is great. Thanks for the support and help @Xon, @jrahmy and @maitikeisi, I'll upload the release to XenForo now.

**Pull request merged**